kilkari/src/main/java/org/motechproject/nms/kilkari/domain/SubscriptionPack.java

```diff
@@ -13,21 +14,26 @@
 public class SubscriptionPack {
 
     @Field
+    @Unique
     private String name;
```

Comment:
Will do. Looking at District as an example, I see that we've set both  Column(allowsNull = "false")  and NotNull. Is our convention to do both? I feel like we've discussed this (and at what layer each constraint is enforced) but I don't remember the outcome.